Makefile

```diff
@@ -83,7 +83,7 @@ $(WEAVEWAIT_EXE) $(SIGPROXY_EXE) $(COVER_EXE) $(RUNNER_EXE):
 	go build -o $@ ./$(@D)
 
 $(WEAVER_UPTODATE): prog/weaver/Dockerfile $(WEAVER_EXE)
-	$(SUDO) docker build -t $(WEAVER_IMAGE) prog/weaver
+	$(SUDO) docker build -t $(WEAVER_IMAGE) -t $(WEAVER_IMAGE):$(WEAVE_VERSION) prog/weaver
```

Comment:
And as you will have discovered by now, if you supply a tag during build then the resulting image won't get tagged as 'latest'.